WordPress/Classes/Services/AccountSettingsService.swift

```diff
@@ -151,6 +151,13 @@ class AccountSettingsService {
         return notificationObserver.map(getSettings).startWith(getSettings())
     }
 
+    func primarySiteNameForSettings(settings: AccountSettings?) -> String {
+        let service = BlogService(managedObjectContext: context)
+        let blog = service.blogByBlogId(settings?.primarySiteID)
+        
+        return blog?.settings?.name ?? String()
```

Comment:
> this is consumed by EditableTextRow, and that class requires String

Sure, I just meant moving the `?? String()` to the calling code in `mapViewModel`, like the rest of the optional fields.